README.md

```diff
@@ -1,20 +1,23 @@
 # iblatlas
-This repository contains tools to manipulate 3D representations of the mouse brain anatomy for electrophysiolgy experiments.
-The tools are mainly using the the Allen Brain Atlas although some other atlases can be used and are written in Python.
+Tools to manipulate hierarchical 3D representations of the mouse brain anatomy for electrophysiolgy experiments.
+The tools are mainly using the the Allen CCF although some other atlases can be used.
 
-One of the requirment of this repository is to use minimal requirements, based of standard matplotlib, numpy and scipy libraries, and excludes visualisation tool.
+**This repository uses minimal requirements, based of standard `matplotlib`, `numpy` and `scipy` libraries, to the exclusion
+of more complex visualization tools such as `pyqt`.**
 
-The documentation can be found here: https://int-brain-lab.github.io/iblenv/notebooks_external/atlas_working_with_ibllib_atlas.html
+## Documentation
+[Full package documentation](https://int-brain-lab.github.io/iblenv/_autosummary/iblatlas.html#module-iblatlas)
+
+[Notebooks and examples](https://int-brain-lab.github.io/iblenv/notebooks_external/atlas_working_with_ibllib_atlas.html)
 
 ## Installation
 `pip install iblatlas`
 
-
 ## Contributing
 Changes are merged by pull requests.
 Release checklist:
-- [ ] Update version in `iblatlas/__init__.py`
-- [ ] Update `CHANGELOG.md`
-- [ ] Create a pull request to the `main` branch on GitHub
+- [x] Update version in `iblatlas/__init__.py`
+- [x] Update `CHANGELOG.md`
+- [x] Create a pull request to the `main` branch on GitHub
 
-Once merged the package is uploaded to PyPI using GitHub Actions.
+Once merged the package is uploaded automatically to PyPI using GitHub Actions.
```

Comment:
Actually this isn't the case. It is the same workflow as ibllib. You need to create a tag with the same version as in the _init_.py and then it is automatically uploaded to pypi once the tag is pushed